themes/default/templates/linkrel;misc;default

```diff
@@ -93,9 +93,15 @@ ELSIF env.script_name.search('journal\.pl');
 
 
 ELSIF env.script_name.search('messages\.pl');
+	IF constants.plugin.Messages;
+		tmp = BLOCK -%]
+<script src="[% constants.real_rootdir %]/messages.js" type="text/javascript"></script>
+		[%- END; linkrels.push(tmp);
+	END;
```

Comment:
Did we decide to load this up on every page because figuring out where we were called from is a bitch and 15 halves.